<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Login</class>
 <widget class="QDialog" name="Login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>345</width>
    <height>450</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>345</width>
    <height>450</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>345</width>
    <height>450</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page">
      <widget class="QLineEdit" name="line_word">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>257</y>
         <width>258</width>
         <height>36</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>258</width>
         <height>36</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>258</width>
         <height>36</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="inputMask">
        <string/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
       <property name="readOnly">
        <bool>false</bool>
       </property>
       <property name="placeholderText">
        <string>请输入密码</string>
       </property>
       <property name="cursorMoveStyle">
        <enum>Qt::LogicalMoveStyle</enum>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_Qrcode">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>387</y>
         <width>51</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>扫码</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>307</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>自动登陆</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QLineEdit" name="line_phoneID">
       <property name="geometry">
        <rect>
         <x>117</x>
         <y>217</y>
         <width>181</width>
         <height>36</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>36</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>207</width>
         <height>36</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="maxLength">
        <number>11</number>
       </property>
       <property name="placeholderText">
        <string>请输入手机号</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_signup">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>387</y>
         <width>51</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>注册</string>
       </property>
      </widget>
      <widget class="QLabel" name="lab_error">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>320</y>
         <width>171</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 0, 0);</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>7</y>
         <width>331</width>
         <height>151</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image: url(:/images/bottom/btn_magnify.png);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_signin">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>337</y>
         <width>262</width>
         <height>40</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>262</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>262</width>
         <height>40</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: #ea4848</string>
       </property>
       <property name="text">
        <string>登陆</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_QRC">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>41</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">  background-color: transparent; </string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>10</y>
         <width>81</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>扫码登录更安全哦</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_countrieslist">
       <property name="geometry">
        <rect>
         <x>56</x>
         <y>218</y>
         <width>59</width>
         <height>36</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>36</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>40</height>
        </size>
       </property>
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">background-repeat: no-repeat;
font: 10pt &quot;Consolas&quot;;
text-align : left;
/*或者通过水平和垂直对齐方式来控制*/
padding-left:2px;  </string>
       </property>
       <property name="text">
        <string>+86</string>
       </property>
       <property name="icon">
        <iconset resource="resource.qrc">
         <normaloff>:/images/btn_down_h.png</normaloff>:/images/btn_down_h.png</iconset>
       </property>
      </widget>
      <widget class="QLabel" name="lab">
       <property name="geometry">
        <rect>
         <x>36</x>
         <y>223</y>
         <width>20</width>
         <height>30</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>20</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>40</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image: url(:/images/phone.png);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="page_2">
      <widget class="QLabel" name="lab_QRC">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>80</y>
         <width>211</width>
         <height>241</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>30</y>
         <width>121</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Consolas</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <italic>false</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(21, 21, 21);</string>
       </property>
       <property name="text">
        <string>扫码登录</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>360</y>
         <width>231</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>使用网易云音乐APP扫码登录</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_backlogin">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>61</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>账号登录</string>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
